packages/workbox-routing/src/Router.ts

```diff
@@ -47,14 +46,15 @@ interface CacheURLsMessageData {
  */
 class Router {
   private readonly _routes: Map<HTTPMethod, Route[]>;
-  private _defaultHandler?: HandlerObject;
+  private readonly _defaultHandler: Map<HTTPMethod, HandlerObject>;
```

Comment:
Naming nit: can we call this `_defaultHandlerMap` (or something like that) since it's no longer a handler type.